lib/routes/movie_details.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:flutter/material.dart';
+import 'package:the_movie_db_flutter/ui_models/movie_preview.dart';
+
+import '../widgets/focus_box.dart';
+import '../widgets/like_button.dart';
+import '../widgets/movie_header.dart';
+import '../widgets/movie_overview.dart';
+import '../widgets/movie_title.dart';
+import '../widgets/vote_average.dart';
+
+class MovieDetails extends StatefulWidget {
+  static const String routeName = "/movie";
```

Comment:
Move the route name to a new consta file that only managed routes like RoutesNames